openwisp_users/admin.py

```diff
@@ -470,7 +470,9 @@ def save_formset(self, request, form, formset, change):
 
 
 class GroupAdmin(BaseGroupAdmin, BaseAdmin):
-    pass
+    def reversion_register(self, model, **kwargs):
+        # Correctly register the proxy model
+        return super().reversion_register(model, for_concrete_model=False, **kwargs,)
```

Comment:
@nemesisdesign I had to include this patch here since we are using a proxy model for Groups. We could instead use an `AbstractGroup` class like we are using for other auth models like User and Organization, but that'd be backward incompatible.

Also, I'm not sure how to test this in this module.